web3_proxy/src/frontend/users/payment.rs

```diff
@@ -436,7 +413,7 @@ pub async fn user_balance_post(
             match NonZeroU64::try_from(recipient.id) {
                 Err(_) => {}
                 Ok(x) => {
-                    app.user_balance_cache.invalidate(&x).await;
+                    app.user_balance_cache.invalidate(&x.get()).await;
```

Comment:
What does .get do here?

Reply:
it gets u64 from NonZeroU64. Unwrapping NonZeroU64 make the code more confuse, so I decided to change the UserBalanceCache to Cache<u64, Balance>, instead of Cache<NonZeroU64, Balance>. Most times user_id is a u64, so it's easier to deal with. Lmk if you think this is an important blocker (i.e. should be changed back to NonZeroU64)

Reply:
It's fine. The types are just getting in the way most of the time. I wish converting was less verbose